doc/api/fs.md

```diff
@@ -1007,6 +1007,39 @@ property indicating whether parent directories should be created. Calling
 `fsPromises.mkdir()` when `path` is a directory that exists results in a
 rejection only when `recursive` is false.
 
+```mjs
+import { mkdir } from 'fs/promises';
+
+try {
+  const projectFolder = new URL('./test/project/', import.meta.url);
+  const createDir = await mkdir(path, { recursive: true });
+
+  console.log(`created ${createDir}`);
+} catch (err) {
+  console.error(err.message);
+}
+```
+
+```cjs
+const { mkdir } = require('fs/promises');
+const { resolve } = require('path');
+
+async function makeDirectory() {
+  try {
+    const projectFolder = resolve('./test/project/');
+    const dirCreation = await mkdir(path, { recursive: true });
+
+    console.log(dirCreation);
+    return dirCreation;
+  } catch (err) {
+    console.error(err.message);
+  }
+
+}
+
+makeDirectory();
```

Comment:
nit: instead of a try/catch, I would put a `.catch(console.error)`: calling an async function without awaiting it or having a catch handler should be frown upon imho.

Reply:
would you want me to move the MJS example to be a function and do the same, or should that continue using top-level await in a try/catch?